script.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python3
+
+import psycopg2
+import requests
+from bs4 import BeautifulSoup
+
+
+def connect():
+    conn = psycopg2.connect(host='db', user="postgres", database="postgres-2", password='mmuntean1!')
+    cur = conn.cursor()
+    cur.execute("SELECT name, description, imdb_id FROM public.movies_movie WHERE movies_movie.description='';")
+    print("The number of parts: ", cur.rowcount)
+    row = cur.fetchone()
+    imdb_ids = []
+
+    while row is not None:
+        row = cur.fetchone()
+        print(row)
+        if row is not None:
+            imdb_ids.append(row[2])
```

Comment:
The cursor has another method to retrieve all the rows. You could use that to avoid looping through the ids twice.